examples/three_layer_constant_fluxes.jl

```diff
@@ -160,7 +174,7 @@ grid = RegularCartesianGrid(size=(Nh, Nh, Nz), x=(0, 512), y=(0, 512), z=(-256,
 Qᵇ = args["buoyancy-flux"]
 Qᵘ = args["momentum-flux"]
 
-prefix = @sprintf("three_layer_constant_fluxes_Qu%.1e_Qb%.1e_Nh%d_Nz%d", abs(Qᵘ), Qᵇ, grid.Nx, grid.Nz)
+prefix = @sprintf("three_layer_constant_fluxes_hr%d_Qu%.1e_Qb%.1e_f%.1e_Nh%d_Nz%d_", hrs, abs(Qᵘ), Qᵇ, f, grid.Nx, grid.Nz)*name
```

Comment:
```suggestion
prefix = @sprintf("three_layer_constant_fluxes_hr%d_Qu%.1e_Qb%.1e_f%.1e_Nh%d_Nz%d_", stop_hours, abs(Qᵘ), Qᵇ, f, grid.Nx, grid.Nz)*name
```

I'm happy to include the stop hours in the filename, but out of curiosity, why is this useful? I intentionally omitted this parameter from the file name because I figured we didn't need to keep data from overlapping simulations that differ only by the stop time.

Reply:
I was thinking this would help prevent overwriting files whose other parameters match, e.g. if your tests run free convection for 0.5 hours but you want the LESbrary simulation to run for 2 days.